docs/adr/0015-use-css-variables-and-scss.md

```diff
@@ -0,0 +1,181 @@
+# Use CSS Variables, Prefer SCSS over CSS-in-JS
+
+- Deciders: Lauren Zugai, Jody Heavener
+- Date: 2020-04-27
+
+Technical Story: [Github issue](https://github.com/mozilla/fxa/issues/4808)
+
+## Context and Problem Statement
+
+The [Settings Redesign](https://github.com/mozilla/fxa/issues/3740) project offers an ideal opportunity to review CSS conventions and architecture for FxA to use not only while building new components for this project but also for FxA moving forward.
+
+This [part 1](https://github.com/mozilla/fxa/issues/4808) of [2](https://github.com/mozilla/fxa/issues/5087) ADR concerns the use of CSS variables, how they come into play with SCSS (vs using SCSS variables), CSS-in-JS considerations, and CSS modules.
+
+Note that 1) "CSS variables" are technically called "CSS custom properties" but will be referred to as the common "variable" nomenclature throughout this ADR and 2) SASS is a CSS preprocessor while SCSS is our preferred syntax of SASS.
+
+## Decision Drivers
+
+- Simplicity and clarity
+- DRY and reusable styles
+- Easy integration and use
+- Current engineering experience and preferences
+
+## Considered Options
+
+### Styling variables
+
+- SASS variables
+- CSS variables
+- A combination of SASS and CSS variables
+
+### Styling placement
+
+- CSS-in-JS with a helper lib like [`styled-components`](https://styled-components.com/) or [`react-jss`](https://cssinjs.org/react-jss/)
+- CSS modules
+- SCSS
+
+## Decision Outcome
+
+Chosen options: "CSS variables" and "SCSS", because:
```

Comment:
Ooh look at that, I actually didn't know that we were using extends in FxA but we are in one or two spots. I'll add extends here, those should be no problem.

Maybe this is a better phrasing for line 39:
> While using CSS variables with SASS will prevent using mixins (or SASS functions like `darken`) that rely on variable values, they can be used without impeding core functionality FxA uses SASS for like nested selectors, extends, and functions and mixins that don't rely on variable values like generation and use of our media queries.

Reply:
Devil's advocate: do we want to encourage `@extend` usage? If used appropriately it can be super beneficial, but I've sometimes found it can become a "junk-drawer" and the resulting selector ends up being this mammoth.